test/test-clusters/src/main/java/org/elasticsearch/test/cluster/FeatureFlag.java

```diff
@@ -16,7 +16,8 @@
  */
 public enum FeatureFlag {
     TIME_SERIES_MODE("es.index_mode_feature_flag_registered=true", Version.fromString("8.0.0"), null),
-    INFERENCE_RESCORER("es.inference_rescorer_feature_flag_enabled=true", Version.fromString("8.10.0"), null);
+    INFERENCE_RESCORER("es.inference_rescorer_feature_flag_enabled=true", Version.fromString("8.10.0"), null),
+    FAILURE_STORE_ENABLED("es.failure_store_feature_flag_enabled=true", Version.fromString("8.10.0"), null);
```

Comment:
I honestly don't know, but should this be changed to 8.11.0 now that we're in that cadence?